packages/core/core/src/Parcel.js

```diff
@@ -174,6 +174,7 @@ export default class Parcel {
 
     let file = {filePath: resolvedPath};
     if (signal && !signal.aborted) {
+      dep.resolvedPath = resolvedPath;
```

Comment:
Not sure about mutating the dep here. Also we might not want to store this in the dep anyway - we could just figure it out based on where it is in the graph. That wouldn't work if we want to cache the resolution though, so perhaps we do want it in the end. Either way, it cannot be on the dep when it is initially created - it will need to be filled in later post-resolution.

Reply:
If we are going to add this field to the dep, it might make more sense to do it in the AssetGraph's updateDependency function.

> That wouldn't work if we want to cache the resolution though

Are talking about the parcel cache or like the in memory "cache" on the v1 Resolver? Caching in memory is pretty much already taken care of by the way the graph is built. If a file is transformed a second time and finds some of the same dependencies it won't try to resolve them again. As for the parcel cache, I'm wondering if we want to cache resolved paths, at least to begin with. Seems like cache invalidation for that could get hairy.

Reply:
I'm talking about the filesystem cache here. In Parcel 1 we do cache the resolved paths, and it increases performance very significantly.